draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -0,0 +1,1568 @@
+   IP/MPLS(-TP), OTN and WDM.  Therefore, this model may be used as a
+   common base model on which other network slicing models, such as
+   [I-D.ietf-ccamp-yang-otn-slicing], may augments with technology-
+   specific constructs.
+
+   As described in Section 3 of
+   [I-D.contreras-teas-slice-controller-models], the data model defined
+   in this document complements the data model defined in
+   [I-D.ietf-teas-ietf-network-slice-nbi-yang].  In addition to the
+   provider's view, the data model defined in this document models the
+   Type 2 service defined in [RFC8453].
+
+   The YANG data model in this document conforms to the Network
+   Management Datastore Architecture (NMDA) [RFC8342].
+
+1.1.  Terminology
+
+   The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+   "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and
+   "OPTIONAL" in this document are to be interpreted as described in BCP
+   14 [RFC2119] [RFC8174] when, and only when, they appear in all
+   capitals, as shown here.
+
+   The terminology for describing YANG data models is found in
+   [RFC7950].
+
+
+
+
+
+
+Liu, et al.              Expires 13 August 2023                 [Page 3]
+
+Internet-Draft      Network Slice Topology Data Model      February 2023
+
+
+1.2.  Tree Diagram
+
+   Tree diagrams used in this document follow the notation defined in
+   [RFC8340].
+
+1.3.  Prefixes in Data Node Names
+
+   In this document, names of data nodes and other data model objects
+   are prefixed using the standard prefix associated with the
+   corresponding YANG imported modules, as shown in Table 1.
+
+             +==========+=======================+===========+
+             | Prefix   | YANG Module           | Reference |
+             +==========+=======================+===========+
+             | yang     | ietf-yang-types       | [RFC6991] |
+             +----------+-----------------------+-----------+
+             | inet     | ietf-inet-types       | [RFC6991] |
+             +----------+-----------------------+-----------+
+             | nt       | ietf-network-topology | [RFC8345] |
+             +----------+-----------------------+-----------+
+             | nw       | ietf-network-topology | [RFC8345] |
+             +----------+-----------------------+-----------+
+             | tet      | ietf-te-topology      | [RFC8795] |
+             +----------+-----------------------+-----------+
+             | te-types | ietf-te-types         | [RFC8776] |
+             +----------+-----------------------+-----------+
+             | ns-topo  | ietf-ns-topo          | RFCXXXX   |
+             +----------+-----------------------+-----------+
+
+             Table 1: Prefixes and Corresponding YANG Modules
+
+   RFC Editor Note: Please replace XXXX with the RFC number assigned to
+   this document.  Please remove this note.
+
+2.  Modeling Considerations
+
+   An IETF network slice is modeled as network topology defined in
```

Comment:
s/ An IETF network slice is modeled / An IETF network slice topology is modeled

Reply:
OK with the changes.